Example/Tests/Other/FetchingLogsTests.swift

```diff
@@ -0,0 +1,63 @@
+//
+// This source file is part of the Web3Swift.io open source project
+// Copyright 2019 The Web3Swift Authors
+// Licensed under Apache License v2.0
+//
+// FetchingLogsTests.swift
+//
+// Created by Vadim Koleoshkin on 14/05/2019
+//
+
+import Nimble
+import Quick
+import SwiftyJSON
+@testable import Web3Swift
+
+class FetchingLogsTests: XCTestCase {
+
+    func testFetchLog() {
+        expect {
+            let receipt: JSON = try TransactionReceiptProcedure(
+                network: MainnetAlchemyNetwork(),
+                transactionHash: BytesFromHexString(
+                    hex: "0x8790901230866dca461f30371f93bd538ab39535bb6c9a57fedbbbdea937ca1e"
+                )
+            ).call()
+            
+            print(receipt["result"]["logs"].arrayValue)
```

Comment:
Let's try not using `print` in tests because prints will contaminate the log and will not be very useful. A better approach would be to attach logs to the running test with an [XCTActivity](https://developer.apple.com/documentation/xctest/xctactivity).

Reply:
as an alternative you can use `guard` and `XCTFail`

Reply:
Thank you for noticing it!